app/scripts/modules/core/pipeline/config/stages/wait/SkipWait.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+
+import {IExecution} from 'core/domain/IExecution';
+import {IExecutionStage} from 'core/domain/IExecutionStage';
+import {ButtonBusyIndicator} from 'core/forms/buttonBusyIndicator/ButtonBusyIndicator';
+import {Application} from 'core/application/application.model';
+import {confirmationModalService} from 'core/confirmationModal/confirmationModal.service';
+import {executionService} from 'core/delivery/service/execution.service';
+import {duration} from 'core/utils/timeFormatters';
+import {OrchestratedItemRunningTime} from 'core/delivery/executionGroup/execution/OrchestratedItemRunningTime';
+
+interface IProps {
+  execution: IExecution;
+  stage: IExecutionStage;
+  application: Application;
+}
+
+interface IState {
+  submitting: boolean;
+  remainingWait?: string;
+}
+
+export class SkipWait extends React.Component<IProps, IState> {
+  private runningTime: OrchestratedItemRunningTime;
```

Comment:
Another confusing case (at the moment, to me). `runningTime` doesn't change (so I guess it goes into `componentDidMount` - copied this from ExecutionMarker), but `remainingWait` does, so it goes in `state`, right?